### PR TITLE
update exporting cloudkms client and functions

### PR DESCRIPTION
**Pull request by ytkang**
## Description
issue:
https://github.com/onflow/flow-go-sdk/issues/201

______

For contributor use:

- [v] Targeted PR against `master` branch
- [v] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [v] Code follows the [standards mentioned here](https://github.com/onflow/flow-go-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [v] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Comment by nanuuki** [comment]
Related to #194 also, as this will allow access to the Client and make it possible to call `Close()`

**Comment by nanuuki** [comment]
@psiemens @turbolent @janezpodhostnik any update on this? :)

File: crypto/cloudkms/cloudkms.go [hunk 1]
```diff
@@ -167,3 +171,11 @@ func parseHashAlgorithm(algo kmspb.CryptoKeyVersion_CryptoKeyVersionAlgorithm) c
 	// https://github.com/onflow/flow-go-sdk/issues/193
 	return crypto.SHA2_256
 }
+
+func ParseSignatureAlgorithm(algo kmspb.CryptoKeyVersion_CryptoKeyVersionAlgorithm) crypto.SignatureAlgorithm {
```
Comment (sideninja) [inline crypto/cloudkms/cloudkms.go:175]: Shouldn't you just change the original function to be exported

File: crypto/cloudkms/cloudkms.go [hunk 2]
```diff
@@ -148,6 +148,10 @@ func (c *Client) GetPublicKey(ctx context.Context, key Key) (crypto.PublicKey, c
 	return publicKey, hashAlgo, nil
 }
 
+func (c *Client) GetKMSClient() *kms.KeyManagementClient {
```
Comment (sideninja) [inline crypto/cloudkms/cloudkms.go:151]: `Get_()` is not a good convention in Go, better to change to `KMSClient()`.

**Comment by sideninja** [comment]
Hi @nanuuki, my apologies, this has slipped under my radar. I've reviewed the PR and left some comments.

**Comment by nanuuki** [comment]
> Hi @nanuuki, my apologies, this has slipped under my radar. I've reviewed the PR and left some comments.

Thank you @sideninja 🙏

**Comment by ytkang** [comment]
@sideninja 
GetKMSClient , ParseSignatureAlgorithm these two functions are already using in my project.
So I can't change at this time.

@nanuuki I hope you to create new code and PR.

**Pull request closed without merging**